src/web/crate_details.rs

```diff
@@ -741,4 +744,42 @@ mod tests {
             Ok(())
         });
     }
+
+    #[test]
+    fn feature_flags_is_hidden_when_empty() {
+        wrapper(|env| {
+            env.fake_release()
+                .name("binary")
+                .version("0.1.0")
+                .binary(true)
```

Comment:
Rather than use this hack, could you change `Fakes::features` to take an `Option`?

Reply:
Done